apps/src/templates/rubrics/RubricContainer.story.jsx

```diff
@@ -91,6 +91,7 @@ const Template = args => (
     teacherHasEnabledAi={false}
     studentLevelInfo={defaultStudentLevelInfo}
     currentLevelName={rubricLevelName}
+    open
```

Comment:
I should've set this prop in https://github.com/code-dot-org/code-dot-org/pull/54153.